src/UI/Components/Diagram/Canvas.test.tsx

```diff
@@ -526,4 +696,57 @@ describe("Canvas.tsx", () => {
     await deleteAndAssert("child-service", 2, 1);
     await deleteAndAssert("child_container", 1, 0);
   });
+
+  // it("deletes links correctly ", async () => {
+  //   //links in this test case cover all of the possible
+  //   const component = setup(mockedInstanceWithReferences);
+  //   render(component);
+
+  //   await screen.findAllByJointSelector("info");
+  //   const connectors = document.querySelectorAll('[data-type="app.Link"]');
+
+  //   expect(connectors).toHaveLength(3);
+
+  //   fireEvent.mouseEnter(connectors[0]);
+  //   //Delay has to be done as library base itself on listeners that are async
+  //   await new Promise((r) => setTimeout(r, 10));
+
+  //   const deleteButton = document.querySelector(
+  //     '[data-tool-name="button"]',
+  //   ) as Element;
+
+  //   await act(async () => {
+  //     await user.click(deleteButton);
+  //   });
+
+  //   const connectors2 = document.querySelectorAll('[data-type="app.Link"]');
+
+  //   expect(connectors2).toHaveLength(2);
+  // });
+
+  it("creates valid object for order_api when creating entity from scratch", () => {});
+  it("creates valid object for order_api when editing entity", () => {});
+
+  // it("zoom-in/zoom-out", async () => {
+  //   // TODO: Resolve "TypeError: viewport.getCTM is not a function"
+  //   const component = setup();
+  //   render(component);
+
+  //   const paper = document.querySelector(".joint-layers") as Element;
+  //   //default value, that is set in the browser
```

Comment:
Is it the same for all browsers?

Reply:
I think i should rename that for "set by library" as it indeed is the same across webpages but thought like why it should be set by browser in the first place :D